templates/security/esm.html

```diff
@@ -116,8 +131,8 @@ <h2>What's covered?</h2>
       <p>ESM continues security updates for:</p>
       <ul class="u-no-margin--bottom">
         <li>high and critical CVEs (Common Vulnerabilities and Exposures), in</li>
-        <li>the Ubuntu base OS and scale-out infrastructure (Ceph, Openstack, see <a href="https://wiki.ubuntu.com/SecurityTeam/ESM/14.04#A14.04_Infrastructure_ESM_Packages" class="p-link--external">the complete list</a>), on</li>
-        <li>the 64-bit x86 architecture.</li>
+        <li>the Ubuntu base OS and scale-out infrastructure</li>
```

Comment:
I've added a [comment](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit?disco=AAAAL7XZpsY) to copy doc about these bullet points I feel like they read weird

Reply:
I removed the `, in` as it does seem like a mistake.